src/plugins/discover/public/application/apps/main/components/view_mode_toggle/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export enum DISCOVER_VIEW_MODE {
+  DOCUMENT_LEVEL = 'documents',
+  FIELD_LEVEL = 'fields',
```

Comment:
We're calling the view "Aggregated view", right? So I think we should call the variables in a similar way.
so how about:

`"AGGREGATED_LEVEL" = 'aggregated'`

because if it's just fields it's not clear what it's meant (without looking it up) IMO
also I think `VIEW_MODE` would be sufficient, since this is already in Discover and not used beyond

Reply:
Updated here https://github.com/elastic/kibana/pull/107184/commits/ba1cad7ff6ee7b3391fb42000306c6b3b96096d5

Reply:
I'm sorry ... now we're calling it "Field Statistics" ... also remember voting for it 🤦 .. So I think it's, more accurate the way it was before `  FIELD_LEVEL = 'fields' ` ... however this change could wait before the merge ... just imagined we would call it "Rolf", and I would again comment to rename it 😃